app/src/main/res/layout/fragment_add_loan_application.xml

```diff
@@ -156,10 +156,6 @@
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"/>
```

Comment:
Align the Submission TextView, with Disbursement fields.

Reply:
That's the problem I faced. I can remove the ImageView but I cannot align them.